documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -121,7 +121,7 @@ In your environment, you might want to use a provided {prodname} container to de
 
 .Prerequisites
 
-* You have the cluster administrator access to an OpenShift cluster.
+* You have Docker installed and rights to create and manage containers.
 * You downloaded the {prodname} connector plug-in(s) that you want to deploy with Avro serialization.
```

Comment:
So that's the case already for the upstream image. Also the sentence "In this procedure, you build a custom Kafka Connect container image for Debezium" before doesn't make sense any longer.

Reply:
The sentence applies for the product, but not the community version, right?